src/webgl/RenderObject.ts

```diff
@@ -0,0 +1,72 @@
+import { Container, DisplayObject } from 'pixi.js'
+import { RenderObjectLifecycle } from '../types'
+
+export default abstract class RenderObject<T extends DisplayObject = DisplayObject> implements RenderObjectLifecycle {
```

Comment:
- `RenderObject` abstract class serves as a base for any display object in an app. i.e. Strokes, Icon, etc. 
- creates a required structure for all display objects making them easy to manage in a generic way
- simple base methods can be overridden or enhanced in practice